libcnb-data/src/buildpackage.rs

```diff
@@ -0,0 +1,92 @@
+use crate::buildpackage::PlatformOs::Linux;
+use serde::{Deserialize, Serialize};
+
+fn platform_default() -> Platform {
+    Platform::default()
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Buildpackage {
+    pub buildpack: BuildpackageUri,
+    #[serde(default)]
+    pub dependencies: Vec<BuildpackageUri>,
+    #[serde(default = "platform_default")]
+    pub platform: Platform,
+}
+
+#[derive(Debug, Deserialize, Serialize, Eq, PartialEq)]
+#[serde(deny_unknown_fields)]
+pub struct BuildpackageUri {
+    pub uri: String,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
+#[serde(deny_unknown_fields)]
+pub struct Platform {
+    // TODO: this should be limited to (linux | windows)
```

Comment:
This TODO seems to be resolved via the `PlatformOs` enum? :)

Reply:
yep